### PR TITLE
USAGOV-725-alt-text-homepage: Alt Text

### PR DESCRIPTION
**Pull request by IsabelLaurenceau**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-725

## Description

Alt text should be available for all images on homepage
You may have to follow https://github.com/usagov/usagov-2021/pull/631 to get the homepage correct
You may also have to add photos to the life experience pages if there are none in the carousel
-- click on each card. On that card's page click edit. Add navigation banner
refresh cache on homepage. images and alt text should be there

## Checklist for the Developer

- [x] A link to the JIRA ticket has been added above.
- [x] The JIRA ticket identifies the desired result of this change.
- [x] The JIRA ticket contains clear acceptance criteria.
- [x] The JIRA ticket contains clear testing steps.
- [x] The JIRA ticket contains a description of "Done".
- [x] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

File: web/themes/custom/usagov/templates/views-view-fields--life-events-view.html.twig [hunk 1]
```diff
@@ -3,29 +3,29 @@
...
+			{% if page_intro != null %}
+				{{page_intro}}
+			{% endif %}
+		{% endif %}
+		{# {{ row.node__field_short_description_field_short_description_value}} #}
+	</p>
+	<div class="thumbnailDiv">
+		{% set image_card_uri = row._entity.field_navigation_banner_image.entity.field_media_image.entity.fileuri %}
+		{% set image_card_alt = row._entity.field_navigation_banner_image.entity.field_media_image.alt %}
+		<img class="thumbnailBanner" alt={{image_card_alt}} src={{file_url(image_card_uri|image_style('max_480w'))}}/>
```
Comment (akf) [inline web/themes/custom/usagov/templates/views-view-fields--life-events-view.html.twig]: The alt text here needs quotation marks around it.

**Review by akf — CHANGES_REQUESTED** [review]
The only problem I see is that the alt text for the Life Events images is missing quotation marks.

**Comment by ghost** [comment]
Reviewed and appears to be working:

USAGOV-725-Test-001

**Review by ghost — APPROVED** [review]
Tested and working in local env.

File: web/themes/custom/usagov/templates/views-view-fields--life-events-view.html.twig (hunk 1 above)
Comment (ghost) [reply web/themes/custom/usagov/templates/views-view-fields--life-events-view.html.twig]: I tested this yesterday, and it appear to add the quotes - see screenshot.

**Pull request merged**